app/src/main/res/values/strings.xml

```diff
@@ -24,12 +24,12 @@
     <string name="home_notice_content">Download Magisk ONLY from the official GitHub page. Files from unknown sources can be malicious!</string>
     <string name="home_support_title">Support Us</string>
     <string name="home_item_source">Source</string>
-    <string name="home_support_content">Magisk is, and always will be, free and open-source. You can however show us that you care by sending a small donation.</string>
+    <string name="home_support_content">Magisk will always be copylefted libre software.\nUse,study, change and share; with all.\nShow you care by sending a donation.</string>
```

Comment:
The rest of the PR is fine. May you preserve the original message here?

Reply:
I find "Free and open-source software"/"Free and Open Source" to be confusing.
Not in the least because of the spelling, but because "free" is not qualified.
In addition (without the Oxford comma) it would mean gratis.
It doesn't thereby explain why it will remain anything.
That is also further, not explained.
"You can however" makes it seem like there is an additional point to stating something beyond "show".

What is it the user should care about, and why is it assumed the user knows without it being explained?

Reply:
I allow even the authorities that be to reconsider, even if I will accept a veto.

Reply:
Consider that part vetoed then. 😛

Reply:
The rest of the PR wasn't fine, and I had given no explanation at that point.
Between changing one error, rewriting some lines, and keeping the entire message, I think consideration for what is in the message is warranted. It can wait till more pressing issues are fixed.